src/util/api.js

```diff
@@ -101,14 +101,9 @@ export const apiGetInstancesForUser = userId => {
 	return handleRequest(methods.GET, `/api/instances/?user=${userId}`)
 }
 
-// TODO update or retire this
-export const apiGetWidgetsByType = (widgetType="default") => {
-	return handleRequest(methods.POST, '/api/widgets/get_by_type/', widgetType )
-}
-
 // Gets widget info
-export const apiGetWidget = (ids=[], type='default') => {
-	let params = `?type=${type}`
+export const apiGetWidget = (ids=[], widgetType='default') => {
```

Comment:
We can probably just ditch "default" as the default param here and use `catalog` instead (or `all`). I saw you included "default" as a supported filter option in the widget API view, which ought to stay regardless.